### PR TITLE
Spackpedia blog post

### PR DESCRIPTION
**Pull request by hauten**
I put together a brief post based on @vsoch tweets about the new Spack Encyclopedia. Happy to modify if needed.

### CODE REVIEW

File: _posts/2021-08-23-spackpedia.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+layout: single
+title: "Introducing the Spackpedia"
+date: 2021-08-23 00:02:00
+tags: spackpedia docs
+---
+
+The Spack community uses a lot of core terms like *spec* and *package* as well as fun ones like *spackage*. In a new twist on documentation, we've created the Spack Encyclopedia, or [Spackpedia](https://spack.github.io/spackpedia/), to help everyone understand our terminology. Click on a term to see a quick definition with an option to read more. Click "read more" for additional details about the term, an example of its usage, a link to Spack documentation, and related terms.
+
+Spackpedia is a [separate repo](https://github.com/spack/spackpedia) within the [Spack organization](https://github.com/spack) on GitHub, and contributions are welcome. Special thanks to Vanessa Sochat and Alec Scott for getting this off the ground.
```
Comment (vsoch) [inline _posts/2021-08-23-spackpedia.md]:
```suggestion
Spackpedia is a [separate repository](https://github.com/spack/spackpedia) within the [Spack organization](https://github.com/spack) on GitHub, and contributions are welcome. Special thanks to Vanessa Sochat and Alec Scott for getting this off the ground.
```

**Review by vsoch — COMMENTED** [review]
Oh I love this, what a treat! I’m pinging @alecbcs so he is also in the loop - this idea came out of one of our fun discussions.

**Comment by hauten** [comment]
So sorry @vsoch - somehow GH isn't notifying me when people reply to or comment on my PRs. So this has languished for ages and may be out of date?

**Comment by vsoch** [comment]
lol! It's okay you can just close it if you like, I don't think anyone cares about the spackpedia. I'm actually not sure how (or why) there are containers being built here, but maybe @tgamblin knows.

**Pull request closed without merging**